tensorflow-logging/src/TensorFlow/Logging.hs

```diff
@@ -64,10 +65,10 @@ import Data.Time.Clock.POSIX (utcTimeToPOSIXSeconds)
 import Lens.Family2 ((.~), (&))
 import Network.HostName (getHostName)
 import Proto.Tensorflow.Core.Framework.Summary (Summary)
-import Proto.Tensorflow.Core.Util.Event (Event, fileVersion, step, summary, wallTime)
+import Proto.Tensorflow.Core.Util.Event (Event, fileVersion, step, summary, wallTime, graphDef)
```

Comment:
Could you move graphDef closer to the front? We try to keep things sorted. Also in the import below.